app/models/alchemy/essence_picture.rb

```diff
@@ -88,12 +88,11 @@ def picture_url_options
     def thumbnail_url(options = {})
       return if picture.nil?
 
-      crop = crop_values_present? || content.settings_value(:crop, options)
-      size = render_size || content.settings_value(:size, options)
+      crop = crop_values_present? && content.settings_value(:crop, options).present?
```

Comment:
There is a feature that you're possibily inadvertently disabling here: When cropping is enabled for a content, the thumbnail/preview image in the element editor should reflect the default cropping and crop settings. This is why a croppable image can be displayed cropped even if no  `crop_values` are present.

Reply:
`content.settings_value(:crop, options)` should always be `true`, `false`, or   `nil`. Using `present?` here is unnecessary.

Reply:
if cropping is enabled and size is present in the element options and you select an image in the element editor, the resize process crops the image as before according to my test.